### ISSUE TITLE
Test routes & features

### ISSUE THREAD

**Issue by Axkaban** [KovDimaY/Codecademy-TDD#2]
Technically you will not need to fill in the `seedItemToDatabase()` method since in utils, it has already `builtItenObject()` [https://github.com/KovDimaY/Codecademy-TDD/blob/master/test/test-utils.js#L6-L11](https://github.com/KovDimaY/Codecademy-TDD/blob/master/test/test-utils.js#L6-L11), that would handle it for you.

Nice delete route [https://github.com/KovDimaY/Codecademy-TDD/blob/master/test/routes/delete-item-test.js#L14](https://github.com/KovDimaY/Codecademy-TDD/blob/master/test/routes/delete-item-test.js#L14)

Very good update and error handling [https://github.com/KovDimaY/Codecademy-TDD/blob/master/test/routes/update-test.js#L27](https://github.com/KovDimaY/Codecademy-TDD/blob/master/test/routes/update-test.js#L27)

**Comment by KovDimaY** [comment]
Yes, it is true, thank you! I just wanted to be sure that two items are different and it works correctly for different objects... :) 
Thank you for the comment!
